test/gurka/test_search_filter.cc

```diff
@@ -91,6 +91,22 @@ TEST_F(SearchFilter, PreferredSide) {
   gurka::assert::osrm::expect_steps(result, {"AB", "AD", "CD", "BC"});
   gurka::assert::raw::expect_path(result, {"AB", "AD", "CD", "BC"});
 }
+TEST_F(SearchFilter, StreetSideCutoff) {
+  auto from = "7";
+  auto to = "8";
+
+  const std::string& request =
+      (boost::format(
+           R"({"locations":[{"lat":%s,"lon":%s},{"lat":%s,"lon":%s,"preferred_side":"same","street_side_cutoff":"primary"}],"costing":"auto"})") %
```

Comment:
this threw me off a little, since it seems it has no impact compared to the default result. looking at the test for `preferred_side` I see though it's the same request, just without the cutoff, and it takes the southbound path. it's the long-standing bug that without an admin db we default to left-side driving..

can you put a capital comment right above the `map` in `SetUpTestSuite`, since it impacts all tests and is pretty confusing on first sight?

Reply:
Yes, thats little confusing. I've added the comment now. 
Here I just copied the test of the preferred_side setting and demonstrated that its not using the preferred side, since the road class is residential, so a too small class to consider the correct side.